CMakeLists.txt

```diff
@@ -115,7 +115,10 @@ else (LIBSBP_SEARCH_PATH)
   endif()
 endif(LIBSBP_SEARCH_PATH)
 
-include_directories("libswiftnav/include")
+if(EXISTS ${PROJECT_SOURCE_DIR}/libswiftnav)
+  include_directories("${PROJECT_SOURCE_DIR}/libswiftnav/include")
+  add_subdirectory("${PROJECT_SOURCE_DIR}/libswiftnav")
+endif()
```

Comment:
should there be an else clause to indicate the user missed a step when initializing the repo?